Documentation/observability/visibility.rst

```diff
@@ -66,6 +66,26 @@ In order for Cilium to populate the ``INGRESS ENFORCEMENT``, ``EGRESS ENFORCEMEN
 and ``VISIBILITY POLICY`` fields, it must run with ``--endpoint-status=policy``
 to make field values visible.
 
+Security Implications
+---------------------
+
+Monitoring Layer 7 traffic comes with some serious security considerations in
+terms of handling potentially sensitive information, such as usernames,
+passwords, query parameters, API keys, etc.
+
+.. Warning::
```

Comment:
```suggestion
.. warning::
```

You also need to indent the body of your warning, or Sphinx won't know what's part of it. This is why the Netlify preview build currently fails with `ERROR: Content block expected for the "Warning" directive; none found.`.